test_util/src/lib.rs

```diff
@@ -1948,17 +1948,60 @@ pub struct CheckOutputIntegrationTest<'a> {
 }
 
 impl<'a> CheckOutputIntegrationTest<'a> {
-  pub fn run(&self) {
+  pub fn run(self) {
```

Comment:
Actually, I'm going to revert these test_util changes. I ended up not needing them, but I will land them in a separate PR.